src/averageAges.js

```diff
@@ -54,8 +68,16 @@ function calculateWomenAverageAge(people, withChildren) {
  *
  * @return {number}
  */
-function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+function calculateAverageAgeDiff(people, onlyWithSon = false) {
+  const childrens = people
```

Comment:
```suggestion
  const children = people
```
children word is already a plural form, FYI